dev/documentation/en/datamodel-settings.md

```diff
@@ -0,0 +1,80 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Settings - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: "Settings"
+
+Settings are additional property values associated with an instance of an entity type.
+
+## The Purpose of Settings
+
+Settings are an important concept in the PKP data model.
+They support two of the [overall goals](datamodel):
+
+### Localization
+
+If some logical attribute (say, a User's `familyname`) is to be localizable,
+the attribute will be stored in settings.
+
+For example a User from Japan might have one `familyname` in Kanji for the `ja_JP` locale
+and another in Latin characters for the `en_US` locale (and the rest of the world).
+For the same one User object, two settings objects will be created to represent
+the two localized versions of the User `familyname` property.
+
+OJS (etc.) knows about these settings and will create and use them regularly.
+See [Entities](architecture-entities.md) for what localization looks like
+at the PHP level.
+
+
+### Extensibility
```

Comment:
I think the Extensibility should go to the end of the Entities page in the docs. And should link to the example in our plugin guide on how to [add custom fields](https://docs.pkp.sfu.ca/dev/plugin-guide/en/examples-custom-field).